scripts/download_image.py

```diff
@@ -102,11 +102,8 @@ def _do_nothing(*args, **kwargs):
     pass
 
 
-REPORT_TYPE = {
-        'progress': _progress_reporter,
-        'plain': _plain_reporter,
-        'quiet': _quiet_reporter,
-}
+REPORT_TYPE = {'progress': _progress_reporter, 'plain': _plain_reporter,
+               'quiet': _quiet_reporter}
```

Comment:
no that make line length long

```python
REPORT_TYPE = {
    'progress': _progress_reporter,
    'plain': _plain_reporter,
     'quiet': _quiet_reporter,
}
```

Reply:
number of indent for REPORT_TYPE
and change request for TXT_SOURCE

Reply:
Fixed
```
REPORT_TYPE = {
        'progress': _progress_reporter,
        'plain': _plain_reporter,
        'quiet': _quiet_reporter,
}

TXT_SOURCE = {
        'arm-wiki': "armImageWikiURLList.txt",
        'chara-wiki': "charaImageWikiURLList.txt",
        'arm-game': "armImageGameURLList.txt",
        'chara-game': "charaImageGameURLList.txt"
}
```